group/group_test.go

```diff
@@ -139,21 +139,32 @@ func testMarshal(t *testing.T, testTimes int, g group.Group) {
 		test.ReportError(t, I, II)
 	}
 
+	sizes := g.Len()
 	got1 := g.NewElement()
 	got2 := g.NewElement()
 	for i := 0; i < testTimes; i++ {
 		x := g.RandomElement(rand.Reader)
```

Comment:
Should we also have a test for unmarshaling the point at infinity since the length will not match in that case?

Reply:
added now